content/docs/higher-order-components.md

```diff
@@ -155,46 +164,47 @@ function withSubscription(WrappedComponent, selectData) {
     }
 
     render() {
-      // ... and renders the wrapped component with the fresh data!
-      // Notice that we pass through any additional props
+      // ... ומרנדרת את הקומפוננטה העטופה עם המידע החדש
+      // הלאה props-שימו לב שאנחנו מעבירים את כל ה
```

Comment:
```suggestion
      // שימו לב שאנחנו מעבירים את כל ה-props הלאה
```

Reply:
This correction breaks the sentence when viewed in the browser, for some reason in comments the sentence order from right-to-left is different.

Reply:
Can you take the suggested sentence and paste it on a program like Word or even in the browser search bar and see if it shows correctly? In my experience GitHub breaks these sentences but they appear fine in the docs

Reply:
I don't have word (mac) - but in the browser bar it still shows as broken
<img width="489" alt="Screen Shot 2019-08-15 at 10 09 34 pm" src="https://user-images.githubusercontent.com/16330962/63093656-6e27ef80-bfa9-11e9-88d1-75ccaa879bb4.png">

Reply:
In the version I committed it looks like this (can you confirm it's the same in your browser?)
<img width="619" alt="Screen Shot 2019-08-15 at 10 10 20 pm" src="https://user-images.githubusercontent.com/16330962/63093687-8730a080-bfa9-11e9-974b-f7b071e954e5.png">

Reply:
Yes you're right I checked [here](https://deploy-preview-95--he-reactjs.netlify.com/docs/higher-order-components.html) and it displays correctly 👍